src/migrations/20181211215451-ArticleTag.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('ArticleTag', {
+    createdAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    updatedAt: {
+      allowNull: false,
+      type: Sequelize.DATE
+    },
+    ArticleId: {
+      type: Sequelize.UUID,
+      primaryKey: true
+    },
+    TagId: {
```

Comment:
I think this should be camelCase as well, `tagId` instead of `TagId`

Reply:
Thanks, will fix that too